synthetic_example.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+import pandas as pd
+from statsmodels.regression.linear_model import RegressionResults
+
+import doubleml as dml
+from doubleml.datasets import fetch_401K
+
+from sklearn.preprocessing import PolynomialFeatures
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.linear_model import LassoCV
+import statsmodels.api as sm
+
+from scipy.stats import norm
+from scipy.linalg import sqrtm
+
+import patsy
+
+import matplotlib.pyplot as plt
```

Comment:
Please check which imports are in fact used. I think you can remove the following
```
from statsmodels.regression.linear_model import RegressionResults
from doubleml.datasets import fetch_401K
from sklearn.preprocessing import PolynomialFeatures
from sklearn.ensemble import RandomForestRegressor
import statsmodels.api as sm
from scipy.stats import norm
from scipy.linalg import sqrtm
import patsy
```